front50-web/src/main/groovy/com/netflix/spinnaker/front50/controllers/V2PipelineTemplateController.java

```diff
@@ -62,29 +74,44 @@
   }
 
   @RequestMapping(value = "", method = RequestMethod.POST)
-  void save(@RequestBody PipelineTemplate pipelineTemplate) {
-    checkForDuplicatePipelineTemplate(pipelineTemplate.getId());
-    getPipelineTemplateDAO().create(pipelineTemplate.getId(), pipelineTemplate);
+  void save(@RequestParam(value = "version", required = false) String version, @RequestBody PipelineTemplate pipelineTemplate) {
+    String templateId = StringUtils.isNotEmpty(version) ? String.format("%s:%s", pipelineTemplate.getId(), version) : pipelineTemplate.getId();
```

Comment:
should version be set to `latest` if empty?

Reply:
Sure.